doc/users.rst

```diff
@@ -1,602 +1,303 @@
 .. _users:
 
```

Comment:
An introduction is probably useful here. Also I am not sure why this user guide is buried inside docs/users.rst with no link to it in the README of the home folder. When I go to a library's github page, i am expecting to see this guide there (to gauge how much work is needed before I jump into it). We have some duplicate code in the README for setup, is there a plan to update that too?

Reply:
We have a plan now :)

Reply:
We have a plan now :)